api/friendsLists.js

```diff
@@ -1 +1,98 @@
-//friendsLists file - delete this if you're writing this file
\ No newline at end of file
+const express = require("express");
+const friendsRouter = express.Router();
+
+const {
+  addFriends,
+  getUserByUsername,
+  removeFriend,
+  getFriendsByUserId,
+  getPostsByUserId,
+} = require("../db");
+
+const { requireUser } = require("./utils");
+
+friendsRouter.post(
+  "/friends/:friendId",
+  requireUser,
+  async (req, res, next) => {
+    try {
+      const { username } = req.params;
+      const { id: userId } = req.user;
+      const { id: friendsId } = await getUserByUsername(username);
+
+      const addFriend = await addFriends(userId, friendsId);
```

Comment:
Okay now I am confused 🤣 If we are just using two IDs to make a friend request then we just don't need a username at all, right? We have access to current user's ID via req.user & we have access to user2's ID via req.params but we aren't using it properly above